k8s/perftest/common/probate/orchestrator-service.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-orchestrator-service
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: probate-orchestrator-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    name: probate-orchestrator-service
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    version: 1.0.0
+  values:
+    tags:
+      idam-pr: false
+    java:
+      replicas: 2
+      memoryRequests: "512Mi"
+      cpuRequests: "250m"
+      memoryLimits: "2048Mi"
+      cpuLimits: "1500m"
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      image: "hmctspublic.azurecr.io/probate/orchestrator-service:prod-6b14362a"
```

Comment:
may be we should remove mention the image without the commas here and keep all files in sync like below-
image: hmctspublic.azurecr.io/probate/orchestrator-service:latest